datajoint/condition.py

```diff
@@ -61,6 +61,13 @@ def append(self, restriction):
             super().append(restriction)
 
 
+class Top:
```

Comment:
Would `@dataclass` be appropriate? It's new in 3.7, so we didn't have it for other classes before we dropped support for 3.6.

Reply:
Also, there is no good reason to handle `dj.Top` in `condition.py` since this module is dedicated to forming the `WHERE` and `HAVING` clauses for `SELECT` statements.  I would handle `dj.Top` entirely within `expression.py`

Reply:
Like [so](https://github.com/datajoint/datajoint-python/pull/1084/files#diff-27f86ee6bd86a0f1f7ffaa9ae916330d6ec989e73ae0db9215801892ac30989cR66-R70)?

Reply:
I wonder if we should default `order_by="KEY"`